app/library/postgres.py

```diff
@@ -86,6 +95,12 @@ async def get_asset_from_postgres(asset_id: str, session, update_cache=False):
     return response
 
 
+async def get_assets_from_postgres(session):
+    assets = get_assets(session)
+    resp_dict = {"assets": [asset.as_dict() for asset in assets]}
+    return resp_dict
```

Comment:
Specifically, which assets would this get? Is it both scene and object assets?

Reply:
Well neither. Scene doesn’t have assets, and objects are just linked to scenes.
These will return the list of actual assets that we have, like the objects in S3. Like in the sample escape room, this would be the models chair, living room, classroom etc.

Reply:
So when we want to see assets on the frontend, how do we show the admin that a certain asset is a scene asset versus an object asset? Furthermore, in the backend UML diagram, there seems to be a relation between scenes and assets, so don't scenes technically have assets?

Reply:
Clarified on Slack. All good

Reply:
Resolved offline on Slack